### DISCUSSION TITLE
Calculated group

### DISCUSSION THREAD

**Discussion by odeddror** [DaxStudio/DaxStudio#749] [Ideas]
What about adding calculation group capabilities into Dax Studio?

**Comment by marcosqlbi** [comment]
What do you mean, exactly?

**Comment by timothywlim** [comment]
I was wondering the same thing. Like could it aggregate all the January entities and make a sum or average out of it.

**Reply by dgosbell** [reply]
@timothywlim You don't need calculations groups to do aggregations like that you can do a simple query or use an inline measure

@odeddror - can you explain more about what you mean by your request? DAX Studio can reference calculation groups in queries, but I can't think what other functionality we could add in this regard.
